handlers/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
 
+- name: reload_collectd
+  systemd:
+   name: collectd
+   daemon_reload: true
+   state: restarted
```

Comment:
If the actual behavior is to make sure collectd is restarted and systemd has reloaded units from disk, I'm not sure if `reload_collectd` is a good name.

Reply:
About this chroot. For that services are configured, but not enable. E.g. when building an image. Thus enabled is fine, but having reload would cause an error most likely (as the service is not running in the first place).